modules/tinymce/src/plugins/advlist/test/ts/atomic/ListUtilsTest.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarElement } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import { isTableCellNode, isListNode } from 'tinymce/plugins/advlist/core/ListUtils';
+
+describe('atomic.tinymce.plugins.advlist.ListUtilsTest', () => {
+  it('isTableCellNode', () => {
+    assert.isTrue(isTableCellNode(SugarElement.fromTag('td').dom), 'td');
```

Comment:
Is atomic tests supposed to fiddle with the dom? Maybe it has a different meaning now days.

Reply:
To be honest, I'm not sure if it's appropriate to put these in the atomic test. Happy to move it to the browser test and include testing the `inList`.